source/features/hide-markdown-diff.css

```diff
@@ -5,11 +5,30 @@
 
 /* Remove decorations on added/changed content */
 .rgh-hide-markdown-diff .markdown-body :is(ins, .added, .changed) {
-	color: inherit !important;
+	color: var(--color-text-primary) !important;
 	background: none !important;
 	box-shadow: none !important;
 }
 
+.rgh-hide-markdown-diff .markdown-body code {
+	color: var(--color-text-primary) !important;
+	background-color: var(--color-markdown-code-bg) !important;
+	border: none !important;
+}
+
+.rgh-hide-markdown-diff .markdown-body blockquote :is(ins, .added, .changed) {
+	color: var(--color-text-tertiary) !important;
+}
+
+/* Fix duplicated content and missing indentation in changed quote blocks #4035 */
+.rgh-hide-markdown-diff .markdown-body .changed > p:not([class]):not(:empty) {
```

Comment:
This targets `<p>` elements that should be wrapped in `<del>` elements but aren't.

Reply:
Should there be `blockquote` here just to make it as specific to the issue as possible?